app/core/models.py

```diff
@@ -137,7 +149,9 @@ class Project(AbstractBaseModel):
 "Authorization: token [gh_PAT]" \
 https://api.github.com/repos/[org]/[repo]',
     )
-    # current_status_id = models.ForeignKey("status", on_delete=models.PROTECT)
+    current_status_id = models.ForeignKey(
```

Comment:
This should be `current_status`.

The old code was written before we realized Django convention is to drop the `_id` ending for foreign key relations.